src/flamenco/runtime/program/fd_compute_budget_program.c

```diff
@@ -140,7 +140,11 @@ fd_executor_compute_budget_program_execute_instructions( fd_exec_txn_ctx_t * ctx
     ctx->compute_unit_price      = updated_compute_unit_price;
   }
 
+  /* https://github.com/anza-xyz/agave/blob/dc4b9dcbbf859ff48f40d00db824bde063fdafcc/runtime-transaction/src/compute_budget_instruction_details.rs#L84-L93 */
```

Comment:
would be nice to use a tagged permalink

Reply:
this change isnt in any tagged release yet